db/schema.rb

```diff
@@ -83,6 +83,16 @@
 
   add_index "courses_skills", ["course_id", "skill_id"], :name => "index_courses_skills_on_course_id_and_skill_id"
 
+  create_table "departments", :force => true do |t|
+    t.string   "name"
+    t.string   "shortname"
+    t.string   "status"
+    t.integer  "contact_id"
+    t.text     "description"
+    t.datetime "created_at",  :null => false
+    t.datetime "updated_at",  :null => false
+  end
+
```

Comment:
I don't know where "departments" came from, but it should not be in this PR.

The miscellaneous introductions of incorrect schema changes are most likely coming from running migrations in other branches, and not rolling them back when switching branches.  I explained this a bit more in https://github.com/kgf/lims/pull/81#issuecomment-181159214

Editing previous migrations can have a similar effect.

Since this kind of undesirable schema change has happened in two recent PRs -- with this PR being worse than #78 -- I hope you can appreciate the importance of managing these schema changes carefully!  We can discuss further if you have questions.  We'll clean this up in another PR.